src/steps/users/converters.ts

```diff
@@ -20,6 +21,67 @@ interface GetCollectionAsFlattendFieldsParams<T extends GSuiteDataCollection> {
   valueMethod: string;
 }
 
+const TIME_PROPERTY_NAMES = /^\w+((T|_t)ime|(O|_o)n|(A|_a)t|(D|_d)ate)$/;
+function isTimeProperty(property: string): boolean {
+  return TIME_PROPERTY_NAMES.test(property);
+}
+
+type ConvertPropertiesOptions = {
+  /**
+   * Parse properties that are named with date/time-like suffixes into number of
+   * milliseconds since epoch (UNIX timestamp).
+   */
+  parseTime?: boolean;
+};
+
+export function convertCustomSchemas(
+  object: any = {},
+  options: ConvertPropertiesOptions = {},
+  prevKey: string = '',
+  converted = {},
+) {
```

Comment:
A couple notes on the interface:
- for `object: any = {}` - adding the ` = {}` makes typescript believe that this is an optional argument. However, this should be a required argument - I would change this to `object: any`.
- It doesn't look like we're actually using `options` in the wild, so I would remove it altogether. It'll simplify the interface.
- Passing in arguments and mutating them can sometimes cause unexpected behaviors - it's always preferable to use "pure" functions. In this case, I would declare `const converted` inside of the function.

```suggestion
export function convertCustomSchemas(
  object: any,
  prevKey: string = '',
) {
  const converted: { [k: string]: any };
```